decentraland/unity-renderer
# feat: analytic tracking for player emotes

## What does this PR change?

Adds an analytic event when the user triggers an emote via command or key

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch=feat/emote-tracking
2. Press 1~4 keys or type /emote kiss in chat
3. The avatar should animate an expression
4. Go to https://metabase.decentraland.systems/ add a new SQL query and write `select * from explorer_dev.used_emote`
5. Some events should appear (it could have 1h delay)
6. The events should have a param name `id` with the expressionId, and `parcel_location` with the coordinates where you triggered the emote

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
